catalog-scripts/idp-catalog-wizard-github-mono.py

```diff
@@ -0,0 +1,200 @@
+import argparse
```

Comment:
Can we rename the file to idp-catalog-wizard-github-monorepo.py? We'll be sharing these files with customers as of now, so better to name it appropriately.